src/Application/Operations/Bulk/BulkUpdateOperation.php

```diff
@@ -6,15 +6,25 @@
 
 use JeroenG\Explorer\Application\BePrepared;
 use JeroenG\Explorer\Application\Explored;
+use JeroenG\Explorer\Domain\IndexManagement\IndexConfigurationInterface;
 
-class BulkUpdateOperation implements BulkOperationInterface
+final class BulkUpdateOperation implements BulkOperationInterface
 {
     /** @var Explored[] */
     private array $models = [];
 
-    public static function from(iterable $iterable): self
+    private static string $indexName;
```

Comment:
Because it is called in a static function, what do you mean by ' an instance field'?

Reply:
Instance field is a class property. Without the static modifier. just `private string $indexName`.
Using a static field in the constructor makes no sense. Static means that it is the same reference for all instances of `BulkUpdateOperation`. So if you would do something like this:

```php
$a = new BulkIndexOperation('a');
$b = new BulkIndexOperation('b');
```
the value of `private static string $indexName` would be 'b' for both instances. 
Using all static functions in this case  doesn't make sense, because you need instance values.